corehq/apps/locations/static/locations/ko/location.js

```diff
@@ -0,0 +1,33 @@
+// for product and user per location selection
+$(function () {
+    $("#id_users-selected_ids").width(800).height(400).multiselect();
+    $("#id_products-selected_ids").width(800).height(400).multiselect();
```

Comment:
But I don't think 800x400 is a magic number or anything like that - we could probably adjust based on screen size if needed, although my preference would be to switch widgets altogether before doing much stylistic work.